spark/v3.4/spark/src/test/java/org/apache/iceberg/spark/source/TestIcebergSourceTablesBase.java

```diff
@@ -1953,6 +1954,62 @@ public void testTableWithInt96Timestamp() throws IOException {
     }
   }
 
+  @Test
+  public void testSessionConfigSupport() throws IOException {
```

Comment:
do we need to throw IOException here?

Reply:
you are right. unfortunately, my IDEA can not recognize relocated guava classes after upgrading, so I can not benefit from the IDEA check. downgraded to recover.